.github/workflows/deploy-job.yml

```diff
@@ -65,16 +68,15 @@ jobs:
           if [ "${{ secrets.firebase_private_key_base64 }}" != "$(gcloud secrets versions access latest --secret ${{ secrets.firebase_private_key_base64_gcp_secret_name }})" ]; then
             echo "${{ secrets.firebase_private_key_base64 }}" | tr -d '\n' | gcloud secrets versions add ${{ secrets.firebase_private_key_base64_gcp_secret_name }} --data-file=-
           fi
-
       - id: deploy
         uses: google-github-actions/deploy-cloudrun@v1
         with:
           service: ${{ secrets.gcp_cloudrun_service_name }}
           region: ${{ secrets.gcp_cloudrun_service_region }}
           project_id: ${{ secrets.gcp_project_id }}
-          secrets: >-
-            FIREBASE_CLIENT_EMAIL=${{ secrets.firebase_client_email_gcp_secret_name }}:latest,
-            FIREBASE_PROJECT_ID=${{ secrets.firebase_project_id_gcp_secret_name }}:latest,
-            FIREBASE_PRIVATE_KEY_BASE64=${{ secrets.firebase_private_key_base64_gcp_secret_name }}:latest
+          env_vars: >-
+            FIREBASE_CLIENT_EMAIL=${{ secrets.firebase_client_email }},
+            FIREBASE_PROJECT_ID=${{ secrets.firebase_project_id }},
+            FIREBASE_PRIVATE_KEY_BASE64=${{ secrets.firebase_private_key_base64 }}
```

Comment:
Looks like the error start appearing after changing the secrets to env vars and then trying to change it back to secrets. Could you try deploying a version without any env vars and then creating it with the secrets?

I think for email and project id it is fine to leave them as env vars, but the private key should be kept a secret.

If we decide to leave the email and project id as env vars then we can remove them from the `update-secrets` step (and cleanup the `_secret_name` variables).

Reply:
I've updated those now. Let me know if you want to add those secrets back @RomuloVitoi